### ISSUE TITLE
Evt is not support MediaQueryList event

### ISSUE THREAD

**Issue by mongris** [garronej/evt#41]
Im use useEvt in my react app to listening  every DOM events. But it throw exception with follow code:

```
const mql = window.matchMedia('(max-width: 600px)')
useEvt(
  ctx => {
    Evt.from(ctx, mql, 'change').attach(...)
  },
  []
)
```

Will Evt support MediaQueryList event?

**Comment by garronej** [comment]
Ok yes I'll add support for it!

**Comment by garronej** [comment]
@mongris , This is fixed in 2.4.1, thanks for reporting.
